scripts/runShiftLeftScan.sh

```diff
@@ -1,3 +1,16 @@
-# Generates results files in the folder reports/. The ShiftLeftScanReader processes the file: reports/scan-full-report.json
-docker run --rm -e "WORKSPACE=${PWD}" -v ~/.m2:/.m2 -v ${PWD}:/app shiftleft/scan scan --src /app --type java
+#!/usr/bin/env bash
 
+# Check for install/updates at https://github.com/ShiftLeftSecurity/sast-scan
+
+source scripts/requireDocker.sh
+
+benchmark_version=$(scripts/getBenchmarkVersion.sh)
+shiflteft_version="2.0.3" # it's not (yet) possible to get the release version so we just assume it
+result_file="results/Benchmark_$benchmark_version-shiftleftscan-v$shiflteft_version.json"
+
+mkdir -p .shiftleftscan-reports
+
+docker run --rm -e "WORKSPACE=${PWD}" -v ~/.m2:/.m2 -v "$PWD":/app -v "$PWD/.shiftleftscan-reports":/app/reports shiftleft/scan scan --src /app --type java
+mv .shiftleftscan-reports/scan-full-report.json "$result_file"
+
+rm -rf .shiftleftscan-reports
```

Comment:
This script seems to work fine, and produce results in /results that are properly scored.